api/src/Repository/ContentNodeRepository.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Activity;
+use App\Entity\Category;
+use App\Entity\ContentNode;
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\Query\Expr\Join;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method null|ContentNode find($id, $lockMode = null, $lockVersion = null)
+ * @method null|ContentNode findOneBy(array $criteria, array $orderBy = null)
+ * @method ContentNode[]    findAll()
+ * @method ContentNode[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ContentNodeRepository extends ServiceEntityRepository implements CanFilterByUserInterface {
+    use FiltersByCampCollaboration;
+
+    public function __construct(ManagerRegistry $registry, string $entityClass = null) {
+        if (null === $entityClass) {
+            parent::__construct($registry, ContentNode::class);
+        } else {
+            parent::__construct($registry, $entityClass);
+        }
+    }
```

Comment:
Man kann gleich den default im parameter angeben.
Ist kürzer, und wird in der Code Completion/Methodenbeschreibung in der IDE gleich angezeigt
```suggestion
      public function __construct(ManagerRegistry $registry, string $entityClass = ContentNode::class) {
        parent::__construct($registry, $entityClass);
    }
```

Reply:
Yep, definitiv schöner. Done.